gateway/cli/src/main.rs

```diff
@@ -92,6 +93,40 @@ pub enum Commands {
         #[clap(long)]
         network: Option<bitcoin::Network>,
     },
+    #[command(subcommand)]
+    Lightning(LightningCommands),
+}
+
+#[derive(Subcommand)]
+pub enum LightningCommands {
```

Comment:
Maybe add a comment here that this API should be intentionally kept very thin

Reply:
Done.